strato

```diff
@@ -175,11 +176,13 @@ then
 else
   if ! docker compose version &> /dev/null
   then
-    docker_compose="docker-compose -p strato --log-level ERROR"
+    marketplace_enabled=$(if [[ ${DISABLE_MARKETPLACE} = true ]]; then echo ""; else echo "--profile marketplace"; fi)
```

Comment:
I would like to suggest the following bash-style syntax to replace this line:
```
[ "$DISABLE_MARKETPLACE" != "true" ] && MP_PROFILE_STRING="--profile marketplace"
```
and then you use ${MP_PROFILE_STRING} to replace ${marketplace_enabled} in docker-compose command on next line